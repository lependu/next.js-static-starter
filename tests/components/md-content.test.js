/* eslint-env jest */
import renderer from 'react-test-renderer'
import MdContent from 'components/md-content'

describe('Components | md-content', () => {
  test('matches snapshot', () => {
    let source = '# Alpha'
    let wrapper = renderer.create(<MdContent source={source} />)
    let tree = wrapper.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
