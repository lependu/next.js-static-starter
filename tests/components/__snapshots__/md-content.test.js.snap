// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components | md-content matches snapshot 1`] = `
<div>
  <h1>
    Alpha
  </h1>
</div>
`;
